cpu/native/can/candev_linux.c

```diff
@@ -7,7 +7,7 @@
  */
 
 /**
- * @ingroup     native_cpu
+ * @ingroup     candev_linux
```

Comment:
C-files aren't rendered by our doxygen-config anyways (so there are no breadcrumbs), and @smlng made a move to do that grouping for C-files.